src/components/Quiz/Quiz.js

```diff
@@ -4,111 +4,132 @@ import { Link } from 'react-router-dom';
-            />
-            False
-          </label>
-          <div>
-            { step > 0 && <button className="" id="previous-button" value="previous" onClick={event => changeQuestion(event)}>Previous</button>}
-            <button
-              className=""
-              id="next-button"
-              value="next"
-              onClick={event => changeQuestion(event)}
-              disabled={buttonDisabled}
-            >
-              Next
-            </button>
-          </div>
+    <section className="quiz-container">
+      <h2 className="quiz-title">Myth Busting Quiz</h2>
+      { questions[questionNumber] ? (
+        <>
+          <form className="quiz-form">
+            <h2>{questions[questionNumber].question}</h2>
+            <div className="inputs-container">
+              <input
+                className="true-input"
+                type="radio"
+                name="answer"
+                checked= { userAnswers[questionNumber] === true }
+                value="true"
+                id="true"
+                onChange={() => handleChange(true)}
+                disabled={formDisabled}
+              />
+              <label htmlFor="true" className="true-label">
+                True
+              </label>
+              <input
+                className="false-input"
+                type="radio"
+                name="answer"
+                checked= { userAnswers[questionNumber] === false }
+                value="false"
+                id="false"
+                onChange={() => handleChange(false)}
+                disabled={ formDisabled || userAnswers[questionNumber] }
+              />
+              <label htmlFor="false" className="false-label">
+                False
+              </label>
+            </div>
+            <div className="arrows-container">
+              <button
+                className="arrow-button"
+                id="previous-button"
+                value="previous"
+                onClick={event => changeQuestion(event)}
+                disabled={ questionNumber ? buttonDisabled : true }
+              >
+                <span className="material-icons">
+                  arrow_back
+                </span>
+              </button>
+              <button
+                className="arrow-button"
+                id="next-button"
+                value="next"
+                onClick={event => changeQuestion(event)}
+                disabled={buttonDisabled}
+              >
+                <span className="material-icons">
+                  arrow_forward
+                </span>
+              </button>
+            </div>
+            <p>You have completed {userAnswers.length / questions.length * 100}% of the quiz!</p>
+          </form>
           { showResult() }
-          <p>You have completed {userAnswers.length / questions.length * 100}% of the quiz!</p>
-        </form>
+        </>
       )
       :
```

Comment:
This was the longest ternary I've seen!  I honestly never think to use them in this way, and I think it's great.